standalone_examples/docker_ce.py

```diff
@@ -0,0 +1,83 @@
+from pyinfra import host
+from pyinfra.modules import apt, server, python, init
+
+# Standalone example to show how to install Docker CE using
+# https://docs.docker.com/install/linux/docker-ce/ubuntu/
```

Comment:
I think it'd be worth linking to `pyinfra-docker` here as well for a prebuild version of this (https://github.com/Fizzadar/pyinfra-docker).

Reply:
Ok. Will do.